CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Changelog
 
+### 2.13
+
+Release date: 2020-08-04
+
+- Fix: Make password parameters work with the `build` step in Jenkins 2.236 and newer ([JENKINS-62305](https://issues.jenkins-ci.org/browse/JENKINS-62305))
+- Fix: Make the Parameters page of builds triggered by the `build` step always display parameters in the order they are defined on the job for that build ([JENKINS-62483](https://issues.jenkins-ci.org/browse/JENKINS-62483))
+- Documentation: [Add guidelines recommending credentials parameters over password parameters when passing secrets using the `build` step](https://plugins.jenkins.io/pipeline-build-step/)
```

Comment:
Should it be the github plugin page?
```suggestion
- Documentation: [Add guidelines recommending credentials parameters over password parameters when passing secrets using the `build` step](https://github.com/jenkinsci/pipeline-build-step-plugin/)
```

Reply:
I could point to either, maybe GitHub is better because it will always be up-to-date? The plugin page is blank right now but will be updated with the content from GitHub once I do the release.

Reply:
Oh, didn't realize the plugin page would update from the github. I was just confused because it was  blank :)

Reply:
Yeah, I should have mentioned in the PR description that it will be blank until after the release. I think I'll leave it as-is for now, but we can update it in the future if needed.